src/mongo/db/encryption/key_id.h

```diff
@@ -34,6 +34,7 @@ Copyright (C) 2022-present Percona and/or its affiliates. All rights reserved.
 #include <cstdint>
 #include <memory>
 #include <string>
+#include "mongo/base/string_data.h"
```

Comment:
Please add empty line before string_data.h include. Server formatting stadard requires it.